### PR TITLE
feat: spice up contributors pr chart styles

### PR DESCRIPTION
**Pull request by OgDev-01**
## Description
This PR updates the styles of the chart and refactors the approach of data fetching. we now have a blend of serverside data fetching with client. this change removes the loading indicator from the application.

## What type of PR is this? (check all applicable)

- [x] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

### Desktop
image

### Mobile
image

## Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 docs.opensauced.pizza
- [ ] 🍕 dev.to/opensauced
- [ ] 📕 storybook
- [x] 🙅 no documentation needed

### CODE REVIEW

**Comment by bdougie** [comment]
I got busy and did not run this locally yesterday but saw the error this morning. 

Screen Shot 2024-01-04 at 5 56 27 AM

File: components/CIResponsiveLine.tsx [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+            tickPadding: 5,
+            tickRotation: 0,
+            legend: "Pull Requests Merged per Day",
+            legendOffset: -40,
+            legendPosition: "middle",
+          }}
+          axisBottom={{
+            tickSize: 0,
+            format: (value) => {
+              return format(parse(value, "dd/MM/yyyy", new Date()), "MMM d");
```
Comment (bdougie) [inline components/CIResponsiveLine.tsx]:
```suggestion
              return format(parse(value, "MM/dd/yyyy", new Date()), "MMM d");
```
This fixes the build error.

**Comment by bdougie** [comment]
> I got busy and did not run this locally yesterday but saw the error this morning.

I found and fixed the error https://github.com/open-sauced/contributor.info/pull/20/commits/6444669500a6412646dca43755545cceb9207340

**Comment by bdougie** [comment]
Can you check https://deploy-preview-20--contributor-info.netlify.app/open-sauced/app

I get a client side exception

image

**Comment by OgDev-01** [comment]
> Can you check [deploy-preview-20--contributor-info.netlify.app/open-sauced/app](https://deploy-preview-20--contributor-info.netlify.app/open-sauced/app)
> 
> I get a client side exception
> 
> image

This works on my end, but the error may likely be cause by an `undefined` variable/function or an uncaught error from an `async` fetch call. Will take a closer look at this

**Review by bdougie — APPROVED** [review]
This LGTM. I'd love to get this on beta and cut a release, but let me know if you want to continue to polishing this @OgDev-01

**Comment by OgDev-01** [comment]
> but let me know if you want to continue to polishing this @OgDev-01

I'll merge this as is for now. Might do some playing around to refactoring and improve the data visualization during the weekend.

**Pull request merged**